omod/src/main/java/org/openmrs/module/bahmni/ie/apps/Constants.java

```diff
@@ -0,0 +1,5 @@
+package org.openmrs.module.bahmni.ie.apps;
+
+public class Constants {
+    public static final String EXPORT_FAILED = "Could not EXPORT the form";
```

Comment:
You can write it in-line and remove this class since this constant is not used anywhere.

Reply:
We are using that variable in BahmniFormServiceImpl to log the error.